bin/dev

```diff
@@ -1,8 +1,2 @@
-#!/usr/bin/env sh
-
-if ! gem list foreman -i --silent; then
-  echo "Installing foreman..."
-  gem install foreman
-fi
-
-exec foreman start -f Procfile.dev "$@"
+#!/usr/bin/env ruby
+exec "./bin/rails", "server", *ARGV
```

Comment:
Come to think of it, this should probably still attempt to use `Procfile.dev` via `foreman`.

Reply:
Yep, agreed.